tests/framework/web/MultipartFormDataParserTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace yiiunit\framework\web;
+
+use yii\web\MultipartFormDataParser;
+use yiiunit\TestCase;
+
+class MultipartFormDataParserTest extends TestCase
+{
+    public function testParse()
+    {
+        if (defined('HHVM_VERSION')) {
+            static::markTestSkipped('Can not test on HHVM because it does not support proper handling of the temporary files.');
```

Comment:
I do not have an HHVM environment and thus I am unable to track the problem or solve it.
All I can tell it fails on Travis.